src/NewDataList/NewDataList.js

```diff
@@ -124,5 +124,14 @@ module.exports = kind({
 			Spotlight.spot(fv);
 			return true;
 		}
+	},
+	isVisibleChild : function(control){
+		var controls = this.orderedChildren.slice(this.firstFullyVisibleI, this.lastFullyVisibleI + 1);
+		for(var i=0;i<controls.length;i++){			
+			if( control.isDescendantOf(controls[i]) ){
```

Comment:
For style consistency, could you change the white-space to reflect `if (control.isDescendantOf(controls[i])) {`